diff --git a/tools/marvell/bin_hdr/src_phy/a38x/mvHighSpeedTopologySpec-38x.c b/tools/marvell/bin_hdr/src_phy/a38x/mvHighSpeedTopologySpec-38x.c
index 3fb7465..01998e9 100755
--- a/tools/marvell/bin_hdr/src_phy/a38x/mvHighSpeedTopologySpec-38x.c
+++ b/tools/marvell/bin_hdr/src_phy/a38x/mvHighSpeedTopologySpec-38x.c
@@ -100,7 +100,7 @@ SERDES_MAP CustomerBoardTopologyConfig[][MAX_SERDES_LANES] =
 #else
 	{ SATA2,	__3Gbps,	SERDES_DEFAULT_MODE,		MV_TRUE,	MV_FALSE },
 #endif
-	{ SGMII2,	__1_25Gbps,	SERDES_DEFAULT_MODE,		MV_FALSE,	MV_FALSE }
+	{ SGMII2,	__3_125Gbps,	SERDES_DEFAULT_MODE,		MV_FALSE,	MV_FALSE }
 }};
 
 
